docs/reference/cli-commands/start.md

```diff
@@ -22,7 +22,12 @@ any other services it depends on, in the correct order.
 
 ## How it works
 
-When starting a service, Pebble executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly. Assuming the command doesn't exit within that time window, the start is considered successful, otherwise `pebble start` will exit with an error, regardless of the `on-failure` value.
+When starting a service, Pebble executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly.
+
+- If the command doesn't exit within the 1 second window, the start is considered successful.
```

Comment:
How about we change this to the positive:
```suggestion
- If the command stays running for the 1 second window, the start is considered successful.
```

Reply:
I think this is the spirit of what we want to say. But I find this exact wording a bit confusing, because the command is allowed to exit during the 1 second window - we only really care that it's running by the end of the window. Maybe this is too fussy... But how about:

```suggestion
- If the command is running at the end of the 1 second window, the start is considered successful.
```

Reply:
Combining the two comments above, I will change it to "If the command is still running at the end of the 1 second window, the start is considered successful."